migration/src/bin/migrate_to.rs

```diff
@@ -0,0 +1,393 @@
+use clap::{builder::PossibleValuesParser, command, Parser, ValueEnum};
+use migration::{
+    sea_orm::{ConnectOptions, Database, DatabaseConnection, EntityTrait, QueryOrder},
+    MigratorTrait,
+};
+use sea_orm_migration::{prelude::*, seaql_migrations};
+use tracing::{info, Level};
+
+#[derive(Copy, Clone, ValueEnum, Debug)]
+enum Direction {
+    Up,
+    Down,
+}
+
+/// Wraps the SeaORM migration library to provide additional features. Lets
+/// you bring migrations up or down to a particular version. Allows dry-run
+/// of migrations.
+#[derive(Parser, Debug)]
+#[command(about, version)]
+struct Args {
+    #[arg(value_enum)]
+    direction: Direction,
+    #[arg(value_parser = available_migrations())]
+    target_version: String,
+    #[arg(short, long)]
+    dry_run: bool,
+    #[arg(short = 'u', long, env = "DATABASE_URL")]
+    database_url: String,
+}
+
+#[async_std::main]
+async fn main() -> Result<(), Error> {
+    let args = Args::parse();
+    tracing_subscriber::fmt()
+        .with_max_level(Level::DEBUG)
```

Comment:
I'm not certain, but I'm pretty sure this will unconditionally enable _very_ verbose logging. I think it'd be better to use [`tracing_subscriber::EnvFilter`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html) so that we can use `RUST_LOG` to govern this. [Janus' `collect` tool](https://github.com/divviup/janus/blob/427e9f5abf676f88808176df699ba724be0d500d/tools/src/bin/collect.rs#L566) has a pretty compact tracing subscriber we can crib here.

Reply:
It incidentally only results in INFO level logging because none of the code paths use any lower level logging. In any case, agreed with having it governed by `RUST_LOG`.

It looks like this sets it up:
```rust
    tracing_subscriber::fmt()
        .with_env_filter(EnvFilter::from_default_env())
        .init();
```

Let me know if this method misses any nuances, admittedly I find there's a bit of decision paralysis as far as `tracing_subscriber` goes.

Reply:
Yeah, that API is a bit intimidating. IIUC the default formatter should be [`Full`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/format/struct.Full.html) which I think will do the trick here -- and we can always iterate if we find that we want more or less log detail.